### PR TITLE
new memoria repo

### PR DESCRIPTION
**Pull request by lfischer79**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by lfischer79 — COMMENTED** [review]
tried adding underglow strip for led

**Review by lfischer79 — COMMENTED** [review]
adding underglow

**Comment by keyboard-magpie** [comment]
Contains VIA and VIAL content, amongst other issues. Please read and digest the PR checklist. I have unticked this box for you.

File: keyboards/memoria/keymaps/default/config.h
Comment (drashna) [inline keyboards/memoria/keymaps/default/config.h:1]: This file should be removed

File: keyboards/memoria/keyboard.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["C2", "C2", "C2", "C2"],
+        "rows": ["D1", "D1", "D1", "D1"]
```
Comment (drashna) [inline keyboards/memoria/keyboard.json:17]: This is certainly not correct.

File: keyboards/memoria/keyboard.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT": {
```
Comment (drashna) [inline keyboards/memoria/keyboard.json:27]: Neither is this, based on the keymap below

File: keyboards/memoria/keymaps/default/rules.mk [hunk 3]
```diff
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+VIAL_INSECURE = yes
+VIALRGB_ENABLE = yes
```
Comment (drashna) [inline keyboards/memoria/keymaps/default/rules.mk:4]:
```suggestion
```

File: keyboards/memoria/keymaps/default/vial.json
Comment (drashna) [inline keyboards/memoria/keymaps/default/vial.json:1]: Remove

File: keyboards/memoria/rules.mk
Comment (drashna) [inline keyboards/memoria/rules.mk:1]: All of these can be moved to the keyboard.json file. `qmk migrate` or `qmk info -kb (name) -f json` can help with this.

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by lfischer79** [comment]
Tbh I’m new to this. I recently obtained a memoria pcb and would like the led strip in the bottom to work with lighting controls either thru QMK config or vial. Wasn’t trying to step on toes. Would just like the led strip to work. The 3 punch outs are by the usb c port and the strips snake from the port side. Then jump down and snake back. Since I couldn’t find anyone to help me out. I thought id try it myself. This is my first attempt at coding firmware.
________________________________

**Comment by drashna** [comment]
No worries.   It's just that we've never accepted vial config in this repo (since it's not supported), and we've recently switched via support over to a separate repo, under the via's team control.

Additionally, I wanted to make sure that incorrect config was flagged explicitly, to make sure you're aware that it needs updating to match the hardware.

**Comment by lfischer79** [comment]
Well. I’ll do my best to make it right however. I’m not working off of any schematic so it probably won’t be right. I just really want these led strips to work. And tbh I didn’t know id need an approval to try and see if I can get it right. Probably won’t happen but I thought id give it the ole college try
________________________________

**Comment by lfischer79** [comment]
If you can give me any pointers would be great. Even though I don’t know ya. Don’t wanna let ya down
________________________________

**Pull request closed without merging**